<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Image/test.jpg</normaloff>:/Image/test.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>55</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title">
    <string>性别</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rBtnMan">
      <property name="text">
       <string>男</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rBtnWoman">
      <property name="text">
       <string>女</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>67</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title">
    <string>婚否</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>未婚</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>已婚</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>91</width>
     <height>92</height>
    </rect>
   </property>
   <property name="title">
    <string>意向城市</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>北京</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>上海</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cBoxWH">
      <property name="text">
       <string>武汉</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
